common/meta/raft-store/src/sled_key_spaces.rs

```diff
@@ -147,6 +154,38 @@ impl SledKeySpace for TableLookup {
     type V = SeqV<TableLookupValue>;
 }
 
+pub struct Shares {}
+impl SledKeySpace for Shares {
+    const PREFIX: u8 = 12;
+    const NAME: &'static str = "shares";
+    type K = u64;
+    type V = SeqV<ShareInfo>;
+}
+
+pub struct ShareLookup {}
+impl SledKeySpace for ShareLookup {
+    const PREFIX: u8 = 14;
+    const NAME: &'static str = "share-look-up";
+    type K = ShareLookupKey;
+    type V = SeqV<ShareLookupValue>;
+}
+
+pub struct ShareOutbounds {}
+impl SledKeySpace for ShareOutbounds {
+    const PREFIX: u8 = 15;
+    const NAME: &'static str = "share-outbounds";
+    type K = ShareOutboundKey;
+    type V = SeqV<ShareOutboundValue>;
+}
+
+pub struct ShareInbounds {}
+impl SledKeySpace for ShareInbounds {
+    const PREFIX: u8 = 16;
+    const NAME: &'static str = "share-inbounds";
+    type K = ShareInboundKey;
+    type V = SeqV<ShareInboundValue>;
+}
+
```

Comment:
What about using an exclusive prefix range for `share` related metas? E.g. `[20,30)`

To ease debugging :DD

Reply:
good  idea